images/dind/master/ovn-kubernetes-master.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source /usr/local/bin/openshift-dind-lib.sh
+source /data/network-plugin
+source /data/ovn-kubernetes
+
+function ovn-kubernetes-master() {
+  local config_dir=$1
+  local kube_config="${config_dir}/admin.kubeconfig"
+
+  ovnsecret=$(/usr/local/bin/oc --config="${kube_config}" get secrets | grep ovn | tail -1 | awk '{ print $1 }')
+  token=$(/usr/local/bin/oc --config="${kube_config}" describe secret $ovnsecret | grep "token:" | awk '{ print $2 }')
+
+  local master_config="${config_dir}/master-config.yaml"
+  cluster_cidr=$(grep clusterNetworkCIDR ${master_config} | cut -f 4 -d' ')
```

Comment:
do you have `python`? might be better to use `yaml.load()` here -- no guarantee that the value is on the same line

Reply:
done, does that look any better?